hwdb.d/60-keyboard.hwdb

```diff
@@ -1276,6 +1276,16 @@ evdev:atkbd:dmi:bvn*:bvr*:bd*:svnMicro-Star*:pn*A10SC*:*
  KEYBOARD_KEY_f1=f20
  KEYBOARD_KEY_f2=f21
 
+# MSI Modern series
+evdev:atkbd:dmi:bvn*:bvr*:bd*:svnMicro-StarInternational*:pnModern*:*
+ KEYBOARD_KEY_f1=f20                                    # Fn+F5 micmute
+ KEYBOARD_KEY_76=1d                                     # Fn+F4 touchpad, becomes meta+ctrl+toggle
```

Comment:
The test fails:

/build/build/../hwdb.d/60-keyboard.hwdb: 192 match groups, 278 matches, 946 properties
Keycode KEY_1D unknown

is ```1d``` a typo?

Reply:
Yes, it typo, maybe I corrupted it by vim) Thanks for the check and feedback.